tests/system/Database/Live/GroupTest.php

```diff
@@ -40,10 +40,13 @@ public function testGroupBy()
 
     public function testHavingBy()
     {
+        $isANSISQL   = in_array($this->db->DBDriver, ['OCI8'], true);
+        $sumIdColumn = $isANSISQL ? 'SUM("id")' : 'SUM(id)';
+
         $result = $this->db->table('job')
             ->select('name')
             ->groupBy('name')
-            ->having('SUM(id) > 2')
+            ->having($sumIdColumn . ' >', 2)
```

Comment:
I think it is not good practice that using variable in the key of *having* or *where* method of QB.

Reply:
Thanks, response.

Fixed it.

Sure!
Looking at this code makes me want to look for SQL Injection!😂